AutomatedTesting/Gem/PythonCoverage/Code/Source/PythonCoverageEditorSystemComponent.cpp

```diff
@@ -226,16 +225,27 @@ namespace PythonCoverage
             return;
         }
 
-        const auto coverageFile = m_coverageDir / AZStd::string::format("%.*s.pycoverage", AZ_STRING_ARG(testCase));
-
-        // If this is a different python script we clear the existing entity components and start afresh
-        if (m_coverageFile != coverageFile)
+        const auto matcherPattern = AZStd::regex("(.*)::(.*)::(.*)");
```

Comment:
question: Is it possible to make this `constexpr`?

Reply:
Alas, not :(